_resources/port1.0/fetch/mirror_sites.tcl

```diff
@@ -556,218 +482,151 @@ set portfetch::mirror_sites::sites(macports_distfiles) [lsearch -all -glob -inli
     ${ywg.ca}://ywg.ca.distfiles.macports.org/mirror/macports/distfiles/:mirror
 " {:*}]
 
-# To update this list use:
-# $ curl -s http://dev.mysql.com/downloads/mirrors.html | grep -E '>HTTP<' | sed -e 's,.*href="\(.*\)">.*,    \1/Downloads/:nosubdir,g' -e 's,//Downloads/:nosubdir,/Downloads/:nosubdir,g' | sort -u
-# To remove bad mirrors look at this inexpensive output:
-# $ for port in mysql{5,51,55,56} ; do echo "port: ${port}" ; for mirror in $(port distfiles $port | grep -v macports | grep -E "^ *(http|ftp)://") ; do echo $mirror ; curl -sI $mirror | grep -E "(^213|Content-Length)" | sed -e '/Content-Length/ s/.*: //' -e '/213/ s/.* //' ; done ; done
+# MySQL
+# Test ports:
+# $ for port in mysql{5,51,55,56} ; do echo "Port: ${port}" && curl -sI $(port distfiles ${port} | grep -v macports | grep -E "^ *(https)://"); done
 set portfetch::mirror_sites::sites(mysql) {
-    http://artfiles.org/mysql/Downloads/:nosubdir
-    http://ftp.arnes.si/mysql/Downloads/:nosubdir
-    http://ftp.gwdg.de/pub/misc/mysql/Downloads/:nosubdir
-    http://ftp.heanet.ie/mirrors/www.mysql.com/Downloads/:nosubdir
-    http://ftp.iij.ad.jp/pub/db/mysql/Downloads/:nosubdir
-    http://ftp.jaist.ac.jp/pub/mysql/Downloads/:nosubdir
-    http://ftp.ntua.gr/pub/databases/mysql/Downloads/:nosubdir
-    http://ftp.sunet.se/pub/unix/databases/relational/mysql/Downloads/:nosubdir
-    http://gd.tuwien.ac.at/db/mysql/Downloads/:nosubdir
-    http://linorg.usp.br/mysql/Downloads/:nosubdir
-    http://mirror.csclub.uwaterloo.ca/mysql/Downloads/:nosubdir
-    http://mirror.leaseweb.com/mysql/Downloads/:nosubdir
-    http://mirror.switch.ch/ftp/mirror/mysql/Downloads/:nosubdir
-    http://mirror.trouble-free.net/mysql_mirror/Downloads/:nosubdir
-    http://mirrors.dedipower.com/www.mysql.com/Downloads/:nosubdir
-    http://mirrors.dotsrc.org/mysql/Downloads/:nosubdir
-    http://mirrors.ircam.fr/pub/mysql/Downloads/:nosubdir
-    http://mirrors.ukfast.co.uk/sites/ftp.mysql.com/Downloads/:nosubdir
-    http://mirrors.xservers.ro/mysql/Downloads/:nosubdir
-    http://mysql.he.net/Downloads/:nosubdir
-    http://mysql.infocom.ua/Downloads/:nosubdir
-    http://mysql.inspire.net.nz/Downloads/:nosubdir
-    http://mysql.linux.cz/Downloads/:nosubdir
-    http://mysql.mirror.ac.za/Downloads/:nosubdir
-    http://mysql.mirror.kangaroot.net/Downloads/:nosubdir
-    http://mysql.mirrors.arminco.com/Downloads/:nosubdir
-    http://mysql.mirrors.crysys.hit.bme.hu/Downloads/:nosubdir
-    http://mysql.mirrors.hoobly.com/Downloads/:nosubdir
-    http://mysql.mirrors.ovh.net/ftp.mysql.com/Downloads/:nosubdir
-    http://mysql.mirrors.pair.com/Downloads/:nosubdir
```

Comment:
Removing http://mysql.mirrors.pair.com (which currently refuses connections) means that https://trac.macports.org/ticket/25674 can be closed by this PR, so I suggest revising the the commit message to close it.

Reply:
Thanks! Added “Additional tickets:” …incase there are more tickets showing up that we can close at the same time.